modules/402-ingress-nginx/images/proxy-failover-iptables/main.go

```diff
@@ -133,16 +134,22 @@ func main() {
 
 func loop(iptablesMgr *iptables.IPTables) error {
 	resp, err := http.Get("http://127.0.0.1:10254/healthz")
-	defer resp.Body.Close()
+
+	if resp != nil {
```

Comment:
NIT: We usually change the order. If the error is nil, the respo body is always not nil. Thus, it is safe to do:

```go
if err != nil {
  log.Println(err)
  return iptablesMgr.DeleteIfExists("nat", chainName, socketExistsRule...)
}

defer resp.Body.Close()

if resp.StatusCode != http.StatusOK {
  log.Printf("Got %d status code", resp.StatusCode)
  return iptablesMgr.DeleteIfExists("nat", chainName, socketExistsRule...)
}
```

Reply:
fixed.
p.s. sorry, didn't notice your comment and pushed with some extra force :-|